libs/nav/src/tpspace/CParameterizedTrajectoryGenerator.cpp

```diff
@@ -445,6 +445,17 @@ const CParameterizedTrajectoryGenerator::TCollisionCell & CParameterizedTrajecto
 }
 
 /*---------------------------------------------------------------
+					getTPObstacleClearance
+  ---------------------------------------------------------------*/
+const mrpt::math::CMatrix & CParameterizedTrajectoryGenerator::CClearanceGrid::getTPObstacleClearance(
```

Comment:
@chatrasen Returning a "const ref"... it says a lot of your C++ experience, great! ;-)